### DISCUSSION TITLE
Casual Chit-chat

### DISCUSSION THREAD

**Discussion by vintagepc** [vintagepc/MINI404#53] [General]
I DON'T THINK THIS NEEDS ANY EXPLANATION.

![image](https://user-images.githubusercontent.com/53943260/99891906-f2513400-2c3c-11eb-91ea-07c4215e6581.png)![image](https://user-images.githubusercontent.com/53943260/99891849-5fb09500-2c3c-11eb-833a-6b0b3b5e7545.png)![image](https://user-images.githubusercontent.com/53943260/99891866-7b1ba000-2c3c-11eb-804b-427196de95df.png)![image](https://user-images.githubusercontent.com/53943260/99891868-8242ae00-2c3c-11eb-91fd-7bab7657e3ee.png)![image](https://user-images.githubusercontent.com/53943260/101267602-70770580-3728-11eb-97f5-f6258eec8e11.png)![image](https://user-images.githubusercontent.com/53943260/101993374-12967080-3c88-11eb-915a-82a25005cbed.png)![image](https://user-images.githubusercontent.com/53943260/104094225-76975f00-525d-11eb-8bba-1d2388fc085d.png)

TL;DR - CAREFUL WHAT YOU WISH FOR....

**Comment by vintagepc** [comment]
[Getting started guide has been moved to the wiki](https://github.com/vintagepc/MINI404/wiki/Getting-Started)

**Comment by vintagepc** [comment]
All code for the Mini implementation and STM32 lives in hw/arm/prusa for easier management and migration to future qemu versions; this minimizes the impact the project has on the source tree.

**Comment by wavexx** [comment]
heh, I'm setting up a build environment to cross-compile *for* the stm32 as we speak ;), this is going to be fun

**Comment by vintagepc** [comment]
welcome aboard!
